### ISSUE TITLE
Future of this plugin

### ISSUE THREAD

**Issue by Vinzent03** [denolehov/obsidian-git#53]
May I ask @denolehov , what is your plan with this plugin? There were no commits since late December and "fixed" issues are not getting closed. I think this plugin is too important, as it can be not maintained anymore. So my question is, will you merge pull requests and release new versions in the future or is a new plugin needed?
Adding me or someone else as a collaborator would be perfectly fine too.

**Comment by Vinzent03** [comment]
Could you please add a LICENSE?

**Comment by Dzheky** [comment]
> Could you please add a LICENSE?

Are you planning to make a new git plugin?

**Comment by Vinzent03** [comment]
I am working on an improved fork. Pull requests to this plugin would be great to, but I doubt they get merged, because of the activity of the author.

**Comment by denolehov** [comment]
@Vinzent03 thank you for raising this issue. Originally, I've developed this plugin just to test the Obsidian alpha Plugin API and to learn a bit of typescript. I am surprised how useful this plugin became for the community.

Unfortunately, at this very moment, other things take priority over work on this plugin. But I'd like to continue working on this and improve it where I can. 

Thank you for showing interest in this, I have made you a collaborator :)
